### ISSUE TITLE
Question about MatrixCity Street dataset experiments

### ISSUE THREAD

**Issue by BitCalSaul** [SunshineWYC/BlockGaussian#29]
Hello,

I read your paper and found the results on the MatrixCity Street dataset very impressive — especially reaching a PSNR of 25.

I would like to ask a few clarifications:

For the Street dataset, did you use small_city/road_down_vertical/outside, horizon, or the entire set (all)?

How many iterations were used for training?

Would it be possible to provide the pretrained model file?

Thank you very much for your great work!

**Comment by SunshineWYC** [comment]
For the MatrixCity street dataset, the image subset for Block_A street views was constructed following the existing methods.

The results for pre-trained models on other scenes will be available in the coming days. Thank you for your patience.

**Comment by SunshineWYC** [comment]
Now we have uploaded the others optimized scene models, you can find the ply file [here](https://pan.baidu.com/s/1rwxUyQLVqofarSQy9axXzg?pwd=36wa).

**Comment by BitCalSaul** [comment]
Hi @SunshineWYC,

Many thanks for your very prompt and helpful replies — really appreciate the detailed clarifications and for sharing the pretrained models so quickly!

I just have one follow-up question: could you kindly let me know where I can find more details about how the Block_A street dataset subset was constructed? It would be very helpful for us to better reproduce your reported results.

Thanks again for your great work!

**Comment by BitCalSaul** [comment]
Additionally, I noticed a possible issue with the uploaded .ply file (points3D_streetAblockgs.ply). When I load it using a simple script (python read_ply_points.py ...), the reported coordinate ranges are:

```
File: point_cloud_merged.ply
Number of points: 5989675
Coordinate ranges:
  X: [nan, nan]
  Y: [nan, nan]
  Z: [nan, nan]
```


It seems the coordinates are all NaN. Could you kindly check whether the file was uploaded correctly?

**Comment by BitCalSaul** [comment]
Hi @SunshineWYC, thanks again for sharing the pretrained models — really appreciate it!

I tried evaluating the MatrixCity aerial pretrained .ply (≈37M Gaussians) on the official test split (741 images). I loaded all attributes (xyz, scale, rot, feature, opacity). However, I only obtain ~24.x PSNR on the test set. May you kindly give me any hint?

<img width="747" height="110" alt="Image" src="https://github.com/user-attachments/assets/c9f8ebd7-b720-4968-9e52-a2ccf24696f1" />

**Comment by SunshineWYC** [comment]
It's recommended to verify the evaluation scripts to make sure aligned with the repo. In addition, if rendered images are saved as jpg format, the image quality degradation is non-negligible.



---Original---
From: ***@***.***&gt;
Date: Tue, Sep 30, 2025 23:45 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [SunshineWYC/BlockGaussian] Question about MatrixCity Streetdataset experiments (Issue #29)


BitCalSaul left a comment (SunshineWYC/BlockGaussian#29)
 
Hi @SunshineWYC, thanks again for sharing the pretrained models — really appreciate it!
 
I tried evaluating the MatrixCity aerial pretrained .ply (≈37M Gaussians) on the official test split (741 images). I loaded all attributes (xyz, scale, rot, feature, opacity). However, I only obtain ~24.x PSNR on the test set. May you kindly give me any hint?
 Cursor.2025-09-30.10.41.29.png (view on web)

**Comment by SunshineWYC** [comment]
For matrixcity_street scene's view, the initial script is launched in [CityGaussian](https://github.com/Linketic/CityGaussian/blob/V1-original/doc/data_preparation.md#prepare-matrixcity-dataset) repo. Hope it is helpful.

As for our pretrained model points3D_streetAblockgs.ply, thanks for your attention, we will verify it as soon as possible!

**Comment by BitCalSaul** [comment]
Hi @SunshineWYC, thanks a lot for the clarification!
I directly compared the rendered images in memory (not saved as JPG), so image compression should not be the cause. I’ll carefully check the evaluation pipeline, especially around [this part of the script](https://github.com/SunshineWYC/BlockGaussian/blob/f2b95e286ec83bc58e6f12d8dbc38982ca82b950/train.py#L164), to make sure my evaluation is fully aligned with your repo.
